tile/lang/gen_stripe.cc

```diff
@@ -427,13 +427,15 @@ class StripeGenerator {
           int start = (out_shape.dims.size() >= shape.dims.size()) ? 0 : (shape.dims.size() - out_shape.dims.size());
           int idx_offset = out_shape.dims.size() - shape.dims.size();  // can be negative
           for (int i = 0; i < shape.dims.size(); i++) {
+            // TODO: Confirm whether i < start case ever exists
             if (i < start) {
               access.emplace_back(Affine{});
             } else {
               const auto& dim = shape.dims[i];
               if (dim.size > 1) {
                 access.emplace_back(Affine{kernel->idxs[i + idx_offset].name});
               } else {
+                // TODO: confirm that this should be empty Affine rather than affine{0}
```

Comment:
I just looked at this `TODO`, and `Affine{}` and `Affine{0}` should be effectively the same (they work slightly differently under the hood but both represent a constant `0` Affine). Sorry for the confusion in suggesting this might create a new monomial index.